controllers/borrowingsController.js

```diff
@@ -0,0 +1,133 @@
+import Borrowing from "../models/borrowings.model.js";
+import Book from "../models/books.model.js";
+import Member from "../models/members.model.js";
+
+// borrow book
+export async function borrowBook(req, res) {
+    let member = await Member.findAll({where: {member_id: req.body.member_id}});
+    let book = await Book.findAll({where: {book_id: req.body.book_id}});
```

Comment:
Same thing here, consider using `,findOne()` rather than `.findAll()` when it comes to a single value.